src/render/painter.js

```diff
@@ -804,6 +835,23 @@ class Painter {
         }
     }
 
+    prepareDrawProgram(context: Context, program: Program<*>, tileID: ?UnwrappedTileID) {
+        const fog = this.style.fog;
```

Comment:
Unnecessarily nitpicky, really, but maybe move this below the following conditional?

Reply:
Addressed in https://github.com/mapbox/mapbox-gl-js/pull/10564/commits/5b2538b30f14eeea1bac520447625111a68bc13b